docs/platforms/javascript/guides/react/getting-started.mdx

```diff
@@ -0,0 +1,336 @@
+
+```javascript {filename:index.jsx}
+import React from "react";
+import * as Sentry from "@sentry/react";
+
+<Sentry.ErrorBoundary fallback={<p>An error has occurred</p>} showDialog>
+  <App />
+</Sentry.ErrorBoundary>;
+```
+
+<Alert>
+  Alternatively, if you're using a class component, you can wrap your application
+  with `Sentry.withErrorBoundary`. Find out more
+  [here](features/error-boundary/#manually-capturing-errors).
+ </Alert>
+
+</Expandable>
+
+## Sending Logs
+
+[Structured logging](/platforms/javascript/guides/react/logs/) lets users send text-based log information from their applications to Sentry. Once in Sentry, these logs can be viewed alongside relevant errors, searched by text-string, or searched using their individual attributes.
+
+Use Sentry's logger to capture structured logs with meaningful attributes that help you debug issues and understand user behavior.
+
+```javascript
+import * as Sentry from "@sentry/react";
+
+const { logger } = Sentry;
+
+// Send structured logs with attributes
+logger.info("User completed checkout", { 
+  userId: 123, 
+  orderId: "order_456", 
+  amount: 99.99 
+});
+
+logger.error("Payment processing failed", {
+  errorCode: "CARD_DECLINED",
+  userId: 123,
+  attemptCount: 3
+});
+
+// Using template literals for dynamic data
+logger.warn(logger.fmt`Rate limit exceeded for user: ${userId}`);
+```
+
+## Customizing Replays (Optional) 
+
+[Replays](/product/explore/session-replay/web/getting-started/) allow you to see video-like reproductions of user sessions. 
+
+By default, Session Replay masks sensitive data for privacy and to protect PII data. You can modify the replay configurations in your client-side Sentry initialization to show (unmask) specific content that's safe to display.
+
+```javascript {filename:instrument.js}
+import * as Sentry from "@sentry/react";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [
+    Sentry.replayIntegration({
+      // This will show the content of the div with the class "reveal-content" and the span with the data-safe-to-show attribute
+      unmask: [".reveal-content", "[data-safe-to-show]"],
+      // This will show all text content in replays. Use with caution.
+      maskAllText: false,
+      // This will show all media content in replays. Use with caution.
+      blockAllMedia: false,
+    }),
+  ],
+  replaysSessionSampleRate: 0.1,
+  replaysOnErrorSampleRate: 1.0,
+  // ... your existing config
+});
+```
+
+```jsx
+<div className="reveal-content">This content will be visible in replays</div>
+<span data-safe-to-show>Safe user data: {username}</span>
+```
+
+## Custom Traces with Attributes (Optional) 
+
+[Tracing](/platforms/javascript/guides/react/tracing/) allows you to monitor interactions between multiple services or applications. Create custom spans to measure specific operations and add meaningful attributes. This helps you understand performance bottlenecks and debug issues with detailed context.
```

Comment:
Maybe you can add a short info here that we already create a bunch of spans. No need to go into detail here, as it's documented on the Tracing page.